### DISCUSSION TITLE
Desktop App 2022?

### DISCUSSION THREAD

**Discussion by fsennj** [chatwoot/chatwoot#3678] [Self hosted] [answered]
Hi all! Great app!! is there any plany for a desktop app in future with notifications (they dont work on self hosted versions yet only on mobile app)..

thank you

**Comment by vishnu-narayanan** [comment, marked as answer]
Related discussion --> https://github.com/chatwoot/chatwoot/discussions/1839

**Comment by fsennj** [comment]
Thank you very much  I will use this to create my desktop / web app!

Von: Vishnu Narayanan ***@***.***>
Gesendet: Montag, 3. Januar 2022 07:40
An: chatwoot/chatwoot ***@***.***>
Cc: Franz Senn jun. ***@***.***>; Author ***@***.***>
Betreff: Re: [chatwoot/chatwoot] Desktop App 2022? (Discussion #3678)


Related discussion --> #1839<https://github.com/chatwoot/chatwoot/discussions/1839>

—
Reply to this email directly, view it on GitHub<https://github.com/chatwoot/chatwoot/discussions/3678#discussioncomment-1898370>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AOVBDJISBSSJF3PWFKMODRTUUFADNANCNFSM5LCZWQOA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://portalnospam.senn-beteiligung.at/link?id=BCAAAABGd7m3lfMOLOIhx3uNR_1Qav4Ffx8-jQB0torD82ts_OsAAABOoD8FzgvvU69z1_mSfcQXllA5fWDvIA-cXON6xK7dNM91sqxLNfcYzbUE26XWMy2q1fRml2Gpegva3UB98bMctH5Zf7yj_b18KVjtpl41EUgmtKSEmcv5VJ0ag8fFQ6KGRe33Rp3fdLFphzdos9bLRzcWHRZQ-XGeIu1mEZzxHNz5YYVwLKLaSZjeilItcigN2Qcb0Yvp5qc0Z-J_R9EORPG-CVO4xHkK-KyK4VJIXfbxxhobPC_zZYIB5U-UGEPdLGR20GpqUdMZxGcF_wfmt1qd7qRjQteO4vMKysE1Vf7lRVpvCh4zjfd_0>.
